landoapi/api/landings.py

```diff
@@ -0,0 +1,72 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+"""
+Transplant API
+See the OpenAPI Specification for this API in the spec/swagger.yml file.
+"""
+from connexion import problem
+from flask import request
+from landoapi.models.landing import (
+    Landing,
+    LandingNotCreatedException,
+    LandingNotFoundException,
+    RevisionNotFoundException,
+)
+
+
+def land(data, api_key=None):
+    """ API endpoint at /revisions/{id}/transplants to land revision. """
+    # get revision_id from body
+    revision_id = data['revision_id']
+    try:
+        landing = Landing.create(revision_id, api_key)
+    except RevisionNotFoundException:
+        # We could not find a matching revision.
+        return problem(
+            404,
+            'Revision not found',
+            'The requested revision does not exist',
+            type='https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404'
+        )
+    except LandingNotCreatedException:
+        # We could not find a matching revision.
+        return problem(
+            502,
+            'Landing not created',
+            'The requested revision does exist, but landing failed',
```

Comment:
It would be nice if the error message said something like "Please retry your request at a later time."